portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase

```diff
@@ -1319,6 +1319,84 @@ definition {
 		Alert.viewErrorMessage(errorMessage = "Your request failed to complete.");
 	}
 
+	@description = "Assert that items are being properly indexed if they exists both in Production and in a Publication"
+	@priority = 3
+	test ViewItemsAreProperlyIndexedInProductionAndPublication {
+		task ("Create a new tag") {
+			JSONAssettag.addTag(
+				groupName = "Guest",
+				tagName = "tag1");
+		}
+
+		task ("Create a Web Content with a tag") {
+			JSONWebcontent.addWebContent(
+				assetTagNames = "tag1",
+				content = "WC WebContent Content",
+				groupName = "Guest",
+				title = "WC WebContent Title");
+		}
+
+		task ("Add a Widget Page with Asset Publisher") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Test Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Guest",
+				layoutName = "Test Page",
+				widgetName = "Asset Publisher");
+		}
+
+		task ("Select the Web Content Article as Asset Type and assert that 'WC WebContent Title' is being returned by the Asset Publisher") {
+			Refresh();
```

Comment:
Is this necessary? `Navigator.gotoPage` navigates by URL so we shouldn't need to refresh the page to navigate.